boards/arm/hexiwear_k64/doc/index.rst

```diff
@@ -189,12 +189,11 @@ program the `OpenSDA DAPLink Hexiwear Firmware`_. Check that switches SW1 and
 SW2 are **on**, and SW3 and SW4 are **off**  to ensure K64F SWD signals are
 connected to the OpenSDA microcontroller.
 
-Add the argument ``-DOPENSDA_FW=daplink`` when you invoke ``west build`` or
-``cmake`` to override the default runner from J-Link to pyOCD:
+Add the argument ``-DOPENSDA_FW=jlink`` when you invoke ``west build`` to
```

Comment:
It looks like some copy/paste errors were made here. It should still be daplink, not jlink, for example. Gotta be careful with these Kinetis boards -- some of them need to be daplink, some need to be jlink; it's not uniform.

Reply:
Well spotted, thanks.